### PR TITLE
More scheduling debug in federated replica set

### PR DESCRIPTION
**Pull request by mwielgus**
cc: @quinton-hoole @jianhuiz @kubernetes/sig-cluster-federation

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/31512)

### CODE REVIEW

File: federation/pkg/federation-controller/replicaset/replicasetcontroller.go [hunk 1]
```diff
@@ -329,6 +332,23 @@ func (frsc *ReplicaSetController) schedule(frs *extensionsv1.ReplicaSet, cluster
 	for clusterName, replicas := range overflow {
 		result[clusterName] += replicas
 	}
+	if glog.V(2) {
+		buf := bytes.NewBufferString(fmt.Sprintf("Schedule - ReplicaSet: %s/%s\n", frs.Namespace, frs.Name))
+		sort.Strings(clusterNames)
+		for _, clusterName := range clusterNames {
+			cur := current[clusterName]
+			target := scheduleResult[clusterName]
+			fmt.Fprintf(buf, "%s: current: %3d target: %3d", clusterName, cur, target)
```
Comment (wojtek-t) [inline federation/pkg/federation-controller/replicaset/replicasetcontroller.go]: Why "%3d"? 
Personally, I would just write ""%d", but if you really want an integer, probably something bigger, e.g. %5d?

**Comment by wojtek-t** [comment]
Minor nit, other than that lgtm.

**Comment by wojtek-t** [comment]
lgtm

File: federation/pkg/federation-controller/replicaset/replicasetcontroller.go [hunk 2]
```diff
@@ -338,9 +358,9 @@ func (frsc *ReplicaSetController) reconcileReplicaSet(key string) error {
 		return nil
 	}
 
-	glog.Infof("Start reconcile replicaset %q", key)
+	glog.V(1).Infof("Start reconcile replicaset %q", key)
```
Comment (quinton-hoole) [inline federation/pkg/federation-controller/replicaset/replicasetcontroller.go]: @mwielgus @wojtek-t Whooah!  Why is this at V(1)?  Surely V(4) is more appropriate?
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/logging.md

Same goes for the rest of this PR.

Comment (mwielgus) [inline federation/pkg/federation-controller/replicaset/replicasetcontroller.go]: Previously it was V(0) :). I can bump it to V4 - no problem.

Comment (mwielgus) [inline federation/pkg/federation-controller/replicaset/replicasetcontroller.go]: @quinton-hoole Please review https://github.com/kubernetes/kubernetes/pull/31528

**Pull request merged**